Module_01/1_HTML/README.md

```diff
@@ -0,0 +1,209 @@
+
+## Boolean Attributes
+Some attributes can be written without values, these are boolean attributes. They only have one value which is the same as the attribute name. A great example is the `disabled` attribute, if you wanted to create a form with a disabled input field you could do the following.
+
+```
+<input type="text" disabled >
+```
+
+------------------------------------------------------------------------------------------------------
+HTML Document Anatomy
+------------------------------------------------------------------------------------------------------
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My test page</title>
+  </head>
+  <body>
+    <p>This is my page</p>
+  </body>
+</html>
+```
+
+1. `<!DOCTYPE html>`: The doctype. Long long ago, when HTML was new (~1991), doctypes were meant to act as links to the set of rules that the HTML  page had to follow to be considered good HTML. There would be automatic error checking and other useful things. They looked something like this:
+```
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+```
+They're not used so much now, and are more a historical artifact taht needs to be included for everything to work as expected `<!DOCTYPE html>` is the shortest string of characters that counts as a valid doctype.
+2. `<html></html>`: This element wraps all the content on the entire page, sometimes refered to as the root element.
+3. `<head></head>`: This element acts as a container for all the stuff you want to include on the HTML page that isn't the content you are showing to your page's viewers. This includes things like keywords and a page descript that you want to appear in search results. CSS to style the content, which we will learn more about in this module :smile:.
+4. `<meta charset="utf-84">`: This element sets the character set your document should use to UTF-8, which includes most characters from the majority of written languages. There isn't a good reason not to set this, it can help avoid problems down the road.
+5. `<title></title>`: This element sets the title of your page which will appear in the browser tab, and is used to describe the page when you bookmark/favorite it.
+6. `<body></body>`: This element contains all of the content you see rendered on a web page.
+
+
+------------------------------------------------------------------------------------------------------
+Start building your portfolio in HTML
+------------------------------------------------------------------------------------------------------
+Let's start building out a portfolio using what we have learned so far.
+
+First let's build a directory to house our portfolio in your terminal type the following command
+```
+$ mkdir portfolio
+```
+
+Next let's change the `pwd` to `portfolio/`
+```
+$ cd portfolio
+```
+
+And then create a Home page
+```
+$ touch home.html
```

Comment:
Could this be `index.html`? Github pages is setup to detect `index.html` on `master` by default. They could configure it differently, but I think it would be easiest if it just works by default.